pkgs/applications/misc/electrum/default.nix

```diff
@@ -54,6 +54,8 @@ python3Packages.buildPythonApplication rec {
     cp -ar ${tests} $sourceRoot/electrum/tests
   '';
 
+  nativeBuildInputs = [ qt5.wrapQtAppsHook qt5.qtbase ];
```

Comment:
Because qt is optional I'd think this would need to be conditioned on `withQt`?
`qtbase` shouldn't be needed also, especially natively.

Reply:
Done.